core/src/style/pointStyleBuilder.cpp

```diff
@@ -358,44 +323,28 @@ void PointStyleBuilder::labelPointsPlacing(const Line& _line, const glm::vec4& u
             }
             break;
         case LabelProperty::Placement::spaced: {
-            Line interpolatedLine;
-            std::vector<float> allAngles;
+            LineSampler<std::vector<Point>> sampler;
```

Comment:
This template type is for the internal storage of LineSampler, the z value is used for length. So I would tend to use `std::vector<glm::vec3>` here.